lost/src/main/java/com/mapzen/android/lost/internal/FusionEngine.java

```diff
@@ -112,6 +118,10 @@ public FusionEngine(Context context, Callback callback) {
     if (passiveInterval < Long.MAX_VALUE) {
       enablePassive(passiveInterval);
     }
+
+    if (prepLooper) {
+      Looper.loop();
```

Comment:
Using this test project: https://github.com/sarahlensing/gps-test

...I observed these results...

Play services:
1. Intents are delivered 
2. Intents are delivered 
3. Intents are delivered 
4. App service is killed without callbacks to `onDestroy` fired, intents stop being delivered (I think this accurately replicates what happens when the system kills it)

I want to do some more debugging around case 4 to understand what is happening. Initially, I thought that the play service was restarted. Using `adb shell dumpsys activity services FusedLocationService` I can see the play service is still running. And then looking at details dumped I saw that `createTime=-19h33m35s710ms` changed from what it was before I force quit the app. However, I dont think this is what it appears to be because it also changed after I quit the app from the recent menu but intents still continued to be delivered. There is something else going on in this case but I'm not sure what.

LOST:
1. Intents are delivered
2. Intents are delivered
3. App service is killed, Lost service is restarted, intents stop being delivered
4. App and Lost services are killed, intents stop being delivered

I'm going to try starting the Lost service in a different process to see if that changes any of the results.